src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -27,8 +27,8 @@
 #include <openrct2/world/Park.h>
 #include <openrct2/world/Surface.h>
 
-#define CHEATS_MONEY_DEFAULT MONEY(10000, 00)
-#define CHEATS_MONEY_INCREMENT_DIV MONEY(5000, 00)
+#define CHEATS_MONEY_DEFAULT 10000.00_GBP
```

Comment:
Can you turn all defines for money into `constexpr`s?

Reply:
done in 47469c6